.travis.yml

```diff
@@ -13,4 +13,4 @@ install:
   - travis_retry composer install
 
 script:
-  - phpunit
\ No newline at end of file
+  - phpunit -c phpunit.xml --tap --coverage-text
```

Comment:
There is no (and should not be) - phpunit.xml
Do we really want to run test coverage on all commits?

Have a look at what we do here: https://github.com/php-http/message/blob/master/.travis.yml

That is a good solution. But make that a separate PR.